src/components/Nav.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { PrimaryNav, MenuLink, Menu } from "./NavElements";
+import { constants } from "../routes/constants";
```

Comment:
u can call the object routes then u access the props inside -> routes.VOLUNTEER ,routes.HOME .....